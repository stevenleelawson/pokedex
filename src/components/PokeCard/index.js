import React from 'react';
import './styles.css';

const PokeCard = ({name, pic, weight, type}) => {
  return (
    <div className='poke-card'>
      <h3>{name}</h3>
      <p>Weight: {weight}</p>
      <p>Type: {type}</p>
      <img src={pic} />

    </div>
  )
}

export default PokeCard;
