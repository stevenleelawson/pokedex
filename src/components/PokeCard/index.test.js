import React from 'react';
import { shallow } from 'enzyme';
import PokeCard from './index';

describe('PokeCard', () => {
  it('should match snapshot', () => {
    const wrapper = shallow(<PokeCard />)
    expect(wrapper).toMatchSnapshot();
  })
})
