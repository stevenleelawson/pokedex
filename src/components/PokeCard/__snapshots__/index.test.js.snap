// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PokeCard should match snapshot 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <PokeCard />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <h3 />,
        <p>
          Weight: 
        </p>,
        <p>
          Type: 
        </p>,
        <img
          src={undefined}
        />,
      ],
      "className": "poke-card",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": undefined,
        },
        "ref": null,
        "rendered": null,
        "type": "h3",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            "Weight: ",
            undefined,
          ],
        },
        "ref": null,
        "rendered": Array [
          "Weight: ",
          undefined,
        ],
        "type": "p",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            "Type: ",
            undefined,
          ],
        },
        "ref": null,
        "rendered": Array [
          "Type: ",
          undefined,
        ],
        "type": "p",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "src": undefined,
        },
        "ref": null,
        "rendered": null,
        "type": "img",
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <h3 />,
          <p>
            Weight: 
          </p>,
          <p>
            Type: 
          </p>,
          <img
            src={undefined}
          />,
        ],
        "className": "poke-card",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": undefined,
          },
          "ref": null,
          "rendered": null,
          "type": "h3",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              "Weight: ",
              undefined,
            ],
          },
          "ref": null,
          "rendered": Array [
            "Weight: ",
            undefined,
          ],
          "type": "p",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              "Type: ",
              undefined,
            ],
          },
          "ref": null,
          "rendered": Array [
            "Type: ",
            undefined,
          ],
          "type": "p",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "src": undefined,
          },
          "ref": null,
          "rendered": null,
          "type": "img",
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
